abb_hardware_interface/package.xml

```diff
@@ -9,9 +9,11 @@
   <maintainer email="zelenak@picknik.ai">Andy Zelenak</maintainer>
 
   <license>Apache2</license>
+  <license>BSD-3-Clause</license>
```

Comment:
```suggestion
  <license>Apache2</license>
```

Unless you had good reason to add a second license here. Dmitri said to me via email that he doesn't mind changing to Apache2. (@dignakov)

Reply:
That's because I added some code from [abb_robot_driver](https://github.com/ros-industrial/abb_robot_driver/tree/master/abb_robot_cpp_utilities) which is BSD.